defaults/main.yml

```diff
@@ -0,0 +1,333 @@
+---
+###############################################################################
+# TODO: Add the default values in here for any of the converted heat params
+# from tripleo-heat-templates. This serves two purposes: it facilitates
+# testing, and it allows for removal of THT params in the future without
+# affecting the ansible role.
+###############################################################################
+# ContainerCollectdImage has no default set in THT, so container_collectd_image
+# will not be set, but my be passed to this role anyway.
+# container_collectd_image : <undefined>
+
+# ContainerCollectdConfigImage has no default set in THT, so container_collectd_config_image
+# will not be set, but my be passed to this role anyway.
+# container_collectd_config_image : <undefined>
+
+# Corresponds to EndpointMap in THT
+# Mapping of service endpoint -> protocol. Typically set via parameter_defaults
+# in the resource registry.
+endpoint_map: {}
+
+# Corresponds to ServiceData in THT
+# Dictionary packing service data
+service_data: {}
+
+# Corresponds to ServiceNetMap in THT
+# Mapping of service_name -> network name. Typically set via parameter_defaults
+# in the resource registry.  This mapping overrides those in
+# ServiceNetMapDefaults.
+service_net_map: {}
+
+# Corresponds to DefaultPasswords in THT
+default_passwords: {}
+
+# Corresponds to RoleName in THT
+# Role name on which the service is applied
+role_name: 
```

Comment:
Why is this valid? Is it reading it as an empty string?

This should be updated, I guess.